### ISSUE TITLE
CORS proxy used incorrectly for api-partner.spotify.com

### ISSUE THREAD

**Issue by leonmxrtin** [spicetify/spicetify-cli#2954]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: Spotify for macOS (Apple Silicon) 1.2.34.783.g923721d9
- Spicetify version: v2.36.3
```

### 📝 Description

When using CosmosAsync to make requests to api-partner.spotify.com endpoints, the CORS proxy is incorrectly added to the request.

**Comment by rxri** [comment]
For api-partner (pathfinder) you should use `Spicetify.GraphQL`

**Comment by ohitstom** [comment]
> For api-partner (pathfinder) you should use `Spicetify.GraphQL`

off topic but graphql is broken again on .34 another .some error

**Comment by rxri** [comment]
GraphQL works just fine. I used it to fix custom app on 1.2.34. You're using it incorrectly or the route you're trying to access doesn't exist anymore
